src/main/java/org/acra/builder/ReportExecutor.java

```diff
@@ -276,9 +292,11 @@ public void run() {
                 }
                 lastActivityManager.clearLastActivity();
             }
-
-            android.os.Process.killProcess(android.os.Process.myPid());
-            System.exit(10);
+            //prevent process kill if a debugger is attached, as this would kill the whole application
+            if (!Debug.isDebuggerConnected()) {
```

Comment:
Should only be checking for !Debug.isDebuggerConnected() once, one either line#266 or line#296, not both.

If the Debugger is connected, do you just want to not kill the process?
Or do you also not want it to call the defaultExceptionHandler and clear the last Activity